js/ui/dash.js

```diff
@@ -343,7 +343,8 @@ const DashActor = new Lang.Class({
         let layout = new Clutter.BoxLayout({ orientation: Clutter.Orientation.VERTICAL });
         this.parent({ name: 'dash',
                       layout_manager: layout,
-                      clip_to_allocation: true });
+                      clip_to_allocation: true,
+                      visible: false });
```

Comment:
Oops! You're right, I'll put that back with a note to be squashed with the original commit in future rebases